lib/kcerts/BUILD.bazel

```diff
@@ -24,7 +26,18 @@ go_library(
         "@com_github_mitchellh_go_homedir//:go-homedir",
         "@org_golang_x_crypto//ssh",
         "@org_golang_x_crypto//ssh/agent",
```

Comment:
Running `bazel run //:gazelle` knows to use the bazel function `select()` for cross-platform support for this shared library.